packages/renderer/src/lib/kube/KubeApplyYAMLButton.svelte

```
@@ -33,17 +33,17 @@ async function kubeApply(): Promise<void> {
     let objects: KubernetesObject[] = await window.kubernetesApplyResourcesFromFile(contextName, result, namespace);
     if (objects.length === 0) {
       await window.showMessageBox({
-        title: 'Kubernetes',
+        title: 'Apply Kubernetes YAML',
         type: 'warning',
         message: `No resource(s) were applied.`,
```

Biome (2.4.11)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)